runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -456,6 +456,13 @@ public void onChatMessage(ChatMessage event)
 				takeScreenshot(fileName, "Duels");
 			}
 		}
+
+		if (config.screenshotCollectionLogEntries() && chatMessage.startsWith("New item added to your collection log: "))
+		{
+			String entry = chatMessage.substring(51, chatMessage.length() - 6);
```

Comment:
I'd prefer `"New item added to your collection log: "` be made a constant, then this line can be changed to:

```
String entry = Text.removeTags(chatMessage).substring(COLLECTION_LOG_TEXT.length);
```